### ISSUE TITLE
transfer of statsd to its own org and call for maintainers

### ISSUE THREAD

**Issue by mrtazz** [statsd/statsd#649]
Hi!

This first comes with a big apology about the lack of maintenance on this project over the last 2 years. There are many reasons for why this project is where it is today. But the biggest one is definitely the fact that internally at Etsy while we run the newest version in production, the feature set we need and use is basically that from 2013. So it has become harder over time to make sense of a direction for the project and making time to test all the config variants and features and steer the project.

This is why we decided that it doesn't make sense for statsd to live on Etsy's GitHub account anymore but rather become its own independent project. This will hopefully make it easier for anyone to contribute and make the project less dependent on Etsy's needs and direction. I've been talking to the people currently owning [the statsd org](https://github.com/statsd) and we are working together to find a new home for the statsd project there.

This move also comes with a call for maintainers. One of the biggest limitations of the project right now is that all maintainers are from inside Etsy and have very limited experience in running statsd in various different architectures and feature configurations. The move to its own space should make it easier to get a diverse group of maintainers on the project that are able to give the project the attention it deserves.

We are currently planning to complete the move by the end of the year and will add new maintainers on the statsd org after that move. If in the meantime you are genuinely interested in becoming a maintainer, start contributing now by e.g. commenting on issues, suggesting labels, testing PRs, writing PRs. This will help with becoming a maintainer as it

- Provides a history of contributing to the project.
- Gives you an idea of what is involved to be a maintainer, so you will know if it's something you actually are still interested in.

We will figure out the specifics of how to then become a maintainer once the move is done.

Thank you all for staying with the project, contributing, and showing interest over the years!

**Comment by enys** [comment]
Hi @mrtazz, Thanks for the update !

**Comment by VladaPetrovic** [comment]
Good call @mrtazz 👍

**Comment by mrtazz** [comment]
quick update: some process and vacation times made this take a little bit longer. We are in the final stages of getting the org transferred.

@BlueHatbRit awesome, thank you for that!

**Comment by mrtazz** [comment]
transfer is done and the call for maintainers has been moved to a new issue (linked above)
